tests/console/sshd.pm

```diff
@@ -144,9 +155,17 @@ sub run {
     # Restore the /etc/ssh/sshd_config
     assert_script_run 'cp /etc/ssh/sshd_config{_before,}';
 
+    # Kill $ssh_testman to stop all SSH sessions
+    script_run "killall -u $ssh_testman";
+    wait_still_screen 3;
+
     record_info("Restart sshd", "Restart sshd.service");
     systemctl("restart sshd");
 
+    if (script_run("journalctl -b | grep -A6 -B24 'segfault\\|fatal'") == 0) {
+        record_info("journalctl", "Please check the journalctl! Segfault or fatal journal entry detected.\n");
+    }
```

Comment:
How about making that a die?

```suggestion
    die("journalctl", "Please check the journalctl! Segfault or fatal journal entry detected.\n")
      if script_run("journalctl -b | grep -A6 -B24 'segfault\\|fatal'") == 0;
```

But I think there is also https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/known_bugs.pm#L39 which might be enough?

Reply:
I am OK with `die` instead of `record_info` but I would like to wait for more people who actually review this test module in different environments so I keep this open for now.

Regarding the `lib::known_bugs` I don't think that the purpose is comparable. We're not trying to catch known but unknown and first of all hidden bugs. We had previously a segfault in SSH port forwarding where the daemon actually silently restarted itself and so the test didn't detect it.

Reply:
yes, of course, it would not be exactly the same. I wanted to make you aware about the method. Surely it would need extension. I am ok with the die-call here.

Reply:
OK, I made the change to `die()` instead of `record_info()`.